src/webpack.js

```diff
@@ -1,15 +1,18 @@
-const assert = require('assert');
+/* eslint-disable security/detect-object-injection */
+/* eslint-disable security/detect-non-literal-fs-filename */
```

Comment:
Even though this is a development tool and therefore doesn't run in production I would personally still prefer that we not disable these rules for an entire file so that developers are encouraged to consider if each instance is really safe or necessary.

Reply:
I can change this but I think this eslint rules are a little too strict for development tools.

Reply:
Fixed (I now use `eslint-disable-next-line <rule>`

Reply:
Great, thanks!